src/style.css

```diff
@@ -1 +1,90 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  background-color: #e5e5e5;
+}
+
+.header {
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 60px;
+  padding: 0 50px;
+}
+
+.logo__img {
+  height: 40px;
+  width: 40px;
+  display: block;
+  margin: 10px 0;
+}
+
+nav,
+ul,
+li {
+  margin: 0;
+  padding: 0;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  height: 100%;
+  text-transform: uppercase;
+}
+
+.nav__item {
+  margin-right: 20px;
+
+}
+
+.nav__link {
+  display: inline-block;
+  text-decoration: none;
+  color: #000;
+  cursor: pointer;
+  font-size: 12px;
+  line-height: 14px;
+  font-weight: 500;
+  font-style: normal;
+  padding: 23px 0;
+  height: 100%;
+
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+
+}
+
+.is-active:after {
+  content: "";
+  background-color: #00acdc;
+  height: 4px;
+  width: 100%;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  border-radius: 8px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+
+}
+
+.nav__last__child {
```

Comment:
Avoid duplication of styles
It's better to use just pseudoclass here because we need just the last element no matter what is it